flax/nnx/tests/graph_utils_test.py

```diff
@@ -425,61 +425,6 @@ def f_pure(graphdef: nnx.graph.GraphDef[Foo], state):
     assert m2 is m
     assert m2.ref is m2
 
-  def test_call_jit_update(self):
```

Comment:
Why do we delete these tests?

Reply:
We would delete `nnx.call` if we add `Pytree`, these test were moved to `pytree_test.py`.